### PR TITLE
Try to improve test coverage

### PR DESCRIPTION
**Pull request by mucsi96**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,9 +48,9 @@
   "author": "Igor Mucsicska <mucsi_96@yahoo.com>",
   "dependencies": {
     "@types/debug": "^4.1.5",
-    "@types/node-fetch": "^2.5.4",
+    "@types/request": "^2.48.4",
     "debug": "^4.1.1",
-    "node-fetch": "^2.6.0"
+    "request": "^2.88.0"
```
Comment (mucsi96) [inline package.json:52]: Request looks much more stable. Now BrowserStack is green every time.

File: test-env/index.ts [hunk 2]
```diff
@@ -193,10 +209,31 @@ const webDriverHost = process.env.BROWSERSTACK
...
+  const testAppPort = process.env.TEST_APP_PORT;
+
+  testEnv.testName = context?.test
+    ?.titlePath()
+    .join('.')
+    .replace(/ /g, '_');
+
+  await session.refresh();
+  await session.navigateTo(
+    `http://localhost:${testAppPort}/#${testEnv.testName}`
```
Comment (mucsi96) [inline test-env/index.ts:232]: Now we add the hash of test name to the URL so it's much easier to track which test is currently running.

File: package.json [hunk 3]
```diff
@@ -14,10 +14,11 @@
   "scripts": {
     "lint": "eslint src/**/*.ts test/**/*.ts test-env/**/*.ts",
     "typecheck": "tsc --project . --noEmit",
-    "test": "ts-mocha test/**/*.test.ts --timeout 60000",
+    "test": "ts-mocha test/**/*.test.ts --timeout 100000",
     "test:debug": "node --inspect ./node_modules/jest/bin/jest.js --detectOpenHandles",
     "test:coverage": "nyc npm test",
-    "test:chrome": "cross-env BROWSER=chrome npm test",
+    "test:chrome": "cross-env BROWSER=chrome npm run test:coverage",
+    "test:coverage:chrome": "cross-env BROWSER=chrome npm run test:chrome",
```
Comment (spnraju) [inline package.json]: the script names are interchanged?

File: package.json (hunk 1 above)
Comment (spnraju) [reply package.json:52]: Sounds good.

File: test-env/index.ts (hunk 2 above)
Comment (spnraju) [reply test-env/index.ts:232]: makes sense.

**Review by spnraju — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (mucsi96) [reply package.json]: Looks like :)

**Pull request merged**